packages/webdriver/tests/command.test.ts

```diff
@@ -258,18 +282,43 @@ describe('command wrapper result log', () => {
         value: false,
         get log() { return this.value }
     }]
+    
+    beforeEach(() => {
+        delete process.env.KEEP_WDIO_LOGGER
+    })
 
     for (const scenario of scenarios) {
         it(`should ${scenario.title} for ${scenario.command.endpoint.command}`, () => {
             const resultFunction = getRequestCallback(scenario.command.method, scenario.command.path, scenario.command.endpoint) as unknown as mockResponse
             resultFunction({ value: scenario.value })
             expect((log.info as jest.Mock).mock.calls[0][1]).toBe(scenario.log)
+            expect(clearLoggerSpy).not.toHaveBeenCalled()
+            clearLoggerSpy.mockClear()
         })
     }
 
     it('should be no result in log if there is value in response', () => {
         const resultFunction = getRequestCallback(takeScreenshotCmd.method, takeScreenshotCmd.path, takeScreenshotCmd.endpoint) as unknown as mockResponse
         resultFunction({})
         expect((log.info as jest.Mock).mock.calls).toHaveLength(0)
+        expect(clearLoggerSpy).not.toHaveBeenCalled()
+        clearLoggerSpy.mockClear()
+    })
+    
+    it('should call clearLogger on deleteSession cmd', () => {
+        const resultFunction = getRequestCallback(deleteSessionCmd.method, deleteSessionCmd.path, deleteSessionCmd.endpoint) as unknown as mockResponse
+        resultFunction({})
+        expect((log.info as jest.Mock).mock.calls).toHaveLength(0)
+        expect(clearLoggerSpy).toHaveBeenCalledTimes(1)
+        clearLoggerSpy.mockClear()
+    })
+
+    it('should NOT call clearLogger on deleteSession cmd if env var - KEEP_WDIO_LOGGER set', () => {
+        const resultFunction = getRequestCallback(deleteSessionCmd.method, deleteSessionCmd.path, deleteSessionCmd.endpoint) as unknown as mockResponse
+        process.env.KEEP_WDIO_LOGGER = 'true'
+        resultFunction({})
+        expect((log.info as jest.Mock).mock.calls).toHaveLength(0)
+        expect(clearLoggerSpy).not.toHaveBeenCalled()
+        clearLoggerSpy.mockClear()
```

Comment:
This test si not needed and can be incorporated into `should be no result in log if there is value in response`